app/helpers/channels_helper.rb

```diff
@@ -113,4 +114,16 @@ def should_display_verification_token?(channel)
     return false if channel.verified? || channel.details_type != "SiteChannelDetails"
     ["no_txt_records", "token_incorrect_dns", "token_not_found_dns"].include?(channel.verification_details)
   end
+
+  def time_until_transfer(channel)
+    return unless channel.verification_pending? || channel.contest_token.present?
+    contest_timesout_at = channel.contest_timesout_at || channel.contesting_channel.contest_timesout_at
+    contest_already_timed_out = (contest_timesout_at - Time.now) < 0
```

Comment:
Are we concerned about the user's timezone here? Seems like this could be a potential for locale issues.

Reply:
@CoryMcDonald I don't think the users time zone matters in this situation.  The app sets a `contest_timesout_at` which we are counting down to.  The time until the timeout shouldn't change regardless of what time the user is in unless I'm misunderstanding something.